pulp_rpm/tests/functional/api/test_pulpimport.py

```diff
@@ -4,459 +4,394 @@
-        cls.imports_api = ImportersPulpImportsApi(cls.core_client)
-        cls.dist_tree_api = ContentDistributionTreesApi(cls.rpm_client)
-
-        cls.import_repos, cls.export_repos, cls.remotes = cls._setup_repositories(
-            RPM_UNSIGNED_FIXTURE_URL
-        )
-        cls.exporter = cls._create_exporter()
-        cls.export = cls._create_export()
-
-    @classmethod
-    def tearDownClass(cls):
-        """No need for class to clean up, we've already done it."""
-        pass
-
-    def tearDown(self):
-        """Remaining clean up repos/export already gone by now."""
-        for repo in self.import_repos:
-            self.repo_api.delete(repo.pulp_href)
-        delete_orphans()
-
-    def _post_export_cleanup(self):
-        """
-        Remove the exported repos and their content, so we're importing into a 'clean' db.
-        """
-        self._delete_exporter(delete_file=False)
-        for remote in self.remotes:
-            self.remote_api.delete(remote.pulp_href)
-        for repo in self.export_repos:
-            self.repo_api.delete(repo.pulp_href)
-        delete_orphans()
-
-    def test_clean_import(self):
-        """Test an import into an empty instance."""
-        # By the time we get here, setUpClass() has created repos and an export for us
-        # Find the export file, create the importer, delete repos/export, and then let
-        # the import happen
-        filenames = [f for f in list(self.export.output_file_info.keys()) if f.endswith("tar.gz")]
-        importer = self._create_importer()
-        self._post_export_cleanup()
-        existing_repos = self.repo_api.list().count
-        # At this point we should be importing into a content-free-zone
-        import_response = self.imports_api.create(importer.pulp_href, {"path": filenames[0]})
+        import_response = importers_pulp_imports_api_client.create(importer.pulp_href, body)
         task_group = monitor_task_group(import_response.task_group)
-        self.assertEqual(len(self.import_repos) + 1, task_group.completed)
-        for repo in self.import_repos:
-            repo = self.repo_api.read(repo.pulp_href)
-            self.assertEqual(f"{repo.pulp_href}versions/1/", repo.latest_version_href)
-        # We should have the same number of repositories, post-import, than however many we had
-        # post-removal of the export-repos
-        self.assertEqual(self.repo_api.list().count, existing_repos)
 
+        return task_group
 
-class CreateMissingReposImportTestCase(PulpTestCase):
+    return _perform_import
+
+
+@pytest.mark.parallel
+def test_import(
+    pulp_importer_factory, import_export_repositories, perform_import, rpm_repository_api
+):
+    """Test an import."""
+    import_repos, exported_repos = import_export_repositories()
+    importer = pulp_importer_factory(import_repos, exported_repos)
+    task_group = perform_import(importer, import_repos, exported_repos, chunked=True)
```

Comment:
I have added `chunked=True` to the original `test_import` test. The chunked export was not used within the tests at all (https://github.com/pulp/pulp_rpm/blob/85d8c9009b47a557cecc1e9cd919bf1cd4f16f04/pulp_rpm/tests/functional/api/test_pulpimport.py#L191).